### ISSUE TITLE
Some tree's are infinite spawned.

### ISSUE THREAD

**Issue by xboxxxxd** [mtadayz/MTADayZ#86]
Ive noticed a Tree that is duping itself everytime at the point when it falls over,
Some tree's do that, like 6% of the time it happens, and stays at that certain tree.

Greetings.xboxxxxd

**Comment by xboxxxxd** [comment]
A Screenshot of the Infinite Tree's Dupe:
![tree dupe problem](https://cloud.githubusercontent.com/assets/6675843/16813494/8cdcfa98-48e6-11e6-8c50-7ed52371948c.png)

**Comment by ghost** [comment]
Dear xboxxxxd,

Thank you for the report. Thank you for helping us improve MTA DayZ. It's a problem I know, it happens only on certain trees. We just need to remove the tree in question from the list of trees that can be cut to solve it.

Rémi Gaffet

**Comment by 0xCiBeR** [comment]
Any updates on identifing the conflictive trees?

**Comment by ghost** [comment]
Yes. I need to retrieve the ID of the tree to remove it from the list of trees that can be cut. This will be done. I work on a fork of MTA DayZ using principles of DayZ Epoch of the Arma 2 game. When I make changes on my fork, the changes will also be implemented to MTA:DayZ.

**Comment by 1B0Y** [comment]
Old bug, i know. But added debug code just in case while I was doing some optimisation of the script.

Update - debug code added in cf57ee6deba3c1811e19672c1c64f81baac84edd
